plugins/colorschemes/oxocarbon.nix

```
@@ -0,0 +1,27 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.colorschemes.oxocarbon;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options = {
+    colorschemes.oxocarbon = {
+      enable = mkEnableOption "oxocarbon";
+
+      package = helpers.mkPackageOption "oxocarbon" pkgs.vimPlugins.oxocarbon-nvim;
+    };
+  };
+
+  config = mkIf cfg.enable {
+    colorscheme = "oxocarbon";
+    extraPlugins = [cfg.package];
+
+    options = {
+      termguicolors = true;
```

```suggestion
      termguicolors = true;
```

What is the idea behind this ?
If we happen to need it, then it would be better to set it to `mkDefault true`.

---

I'm ngl, I just copied the "one" color scheme and changed some things, as you've probably already guessed. I definetly agree that it would be better to use mkDefault, thanks for catching that.

---

Ok, can you please make the change then ?

---

Sorry, I went to bed. I'll do that right now.